stencil-library/.storybook/preview.jsx

```diff
@@ -13,6 +15,15 @@ export const parameters = {
   actions: { argTypesRegex: "^on.*" },
   docs: {
     extractArgTypes: extractArgTypesFactory({ dashCase: true }),
+    page: () => (
```

Comment:
So does this mean that our stories can also use jsx? Because that would be awesome